src/Umbraco.Web.UI.Client/src/assets/lang/da-dk.ts

```diff
@@ -535,6 +536,7 @@ export default {
 		linkToMedia: 'Link til medie',
 		selectContentStartNode: 'Vælg startnode for indhold',
 		selectMedia: 'Vælg medie',
+		chooseMedia: 'Vælg medie start node',
```

Comment:
`start node` should be a single word in Danish :)

Reply:
Thanks @bjarnef I have fixed it 😄